consensus/src/sync/syncer.rs

```diff
@@ -215,6 +218,12 @@ impl<N: Network, M: MacroSync<N::PeerId>, L: LiveSync<N>> Syncer<N, M, L> {
         let blockchain = self.blockchain.clone();
         let network = Arc::clone(&self.network);
 
+        // Light nodes do not provide accounts chunks, so they are classified as incompatible.
```

Comment:
Maybe move this to the "move_peer_into_live_sync" function, so that light nodes are not moved into live sync?